makefiles/Makefile.docs-datalake

```diff
@@ -17,8 +17,7 @@ SNOOTY_DB_PWD = $(shell printenv MONGO_ATLAS_PASSWORD)
 
 # Parse our published-branches configuration file to get the name of
 # the current "stable" branch. This is weird and dumb, yes.
-STABLE_BRANCH=`grep 'manual' build/docs-tools/data/${PROJEC
-T}-published-branches.yaml | cut -d ':' -f 2 | grep -Eo '[0-9a-z.]+'`
+STABLE_BRANCH=`grep 'manual' build/docs-tools/data/${PROJECT}-published-branches.yaml | cut -d ':' -f 2 | grep -Eo '[0-9a-z.]+'`
```

Comment:
no manual in datalake.

Reply:
manual refers to this key i believe: https://github.com/mongodb/docs-worker-pool/blob/meta/publishedbranches/docs-datalake.yaml#L10